Servers/WindowServer/WSMenuManager.cpp

```diff
@@ -157,3 +163,8 @@ void WSMenuManager::handle_menu_mouse_event(WSMenu& menu, const WSMouseEvent& ev
         return;
     }
 }
+
+void WSMenuManager::set_window_needs_resize()
```

Comment:
Let's improve consistency between the function and member names here.
Either `set_window_needs_resize()` + `m_window_needs_resize`
Or `set_needs_window_resize()` + `m_needs_window_resize`
Both are okay, but choose one :)